scipy/fftpack/tests/test_basic.py

```diff
@@ -324,6 +324,29 @@ def test_invalid_sizes(self):
         assert_raises(ValueError, rfft, [])
         assert_raises(ValueError, rfft, [[1,1],[2,2]], -5)
 
+    # See Issue #5790
```

Comment:
I guess the comment in `basic.py` is enough to explain what's happening, never mind about that.

@pv a quick `git grep` shows that much new code has been using `#<N>` instead. We might want to document the convention of using gh- somewhere.